pkgs/by-name/pr/previewqt/package.nix

```diff
@@ -1,64 +1,98 @@
 {
   lib,
-  vips,
-  resvg,
-  mpv,
-  libraw,
-  imagemagick,
-  libdevil,
-  stdenv,
-  fetchFromGitLab,
   cmake,
+  exiv2,
+  extra-cmake-modules,
+  fetchFromGitLab,
+  imagemagick,
   libarchive,
+  libdevil,
+  libraw,
+  mpv,
+  pkg-config,
   qt6Packages,
-  extra-cmake-modules,
-  exiv2,
+  resvg,
+  stdenv,
+  vips,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "previewqt";
   version = "3.0";
 
   src = fetchFromGitLab {
+    name = "previewqt-sources-${finalAttrs.version}";
     owner = "lspies";
     repo = "previewqt";
-    rev = "refs/tags/v${version}";
+    rev = "refs/tags/v${finalAttrs.version}";
     hash = "sha256-cDtqgezKGgSdhw8x1mM4cZ0H3SfUPEyWP6rRD+kRwXc=";
   };
 
-  # can't find qtquick3d
-  strictDeps = false;
-
   nativeBuildInputs = [
     cmake
-    extra-cmake-modules
+    pkg-config
     qt6Packages.wrapQtAppsHook
   ];
 
-  buildInputs = [
-    exiv2
-    imagemagick
-    qt6Packages.poppler
-    qt6Packages.qtmultimedia
-    qt6Packages.qtquick3d
-    qt6Packages.qtsvg
-    qt6Packages.qttools
-    qt6Packages.qtwebengine
-    libarchive
-    libdevil
-    libraw
-    mpv
-    resvg
-    vips
-  ];
+  buildInputs =
+    [
+      exiv2
+      extra-cmake-modules
+      imagemagick
+      libarchive
+      libdevil
+      libraw
+      mpv
+      resvg
+      vips
+    ]
+    ++ [
```

Comment:
This is ugly in so many levels...

Reply:
Then why add it?

Reply:
Because `nixfmt-rfc-style` forced me to do this...

Reply:
I see a `[ ... ] ++ [ ... ]`, there is not `with x;`

Reply:
Not a serious problem. I will ignore this.